repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -108,6 +108,7 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
         val userId = (json \ "access" \ "user" \ "id").as[String]
         val roles = (json \ "access" \ "user" \ "roles").as[JsArray].value
           .map(jsRole => Role((jsRole \ "name").as[String], (jsRole \ "tenantId").asOpt[String]))
+          .filter(role => !ignoredRoles.contains(role.name))
```

Comment:
You might want to use `filterNot` here? I think it makes it clearer, but it's not like what you have is unclear at all.

`.filterNot(role => ignoredRoles.contains(role.name))`

Reply:
I thought about it, but i find `filterNot` frankly more confusing than just not-ing my condition when it isn't what i want. Those methods always do the opposite of what i think they'll do.

Reply:
Fair enough.